### ISSUE TITLE
Autofill with zeros if not typed

### ISSUE THREAD

**Issue by itajackass** [igorescobar/jQuery-Mask-Plugin#628]
I don't know how to simply autofill my duration field if I not provide all digit:

$('.time').mask('00:00', {placeholder: "__:__", clearIfNotMatch: true});

For example if I type: 02 only, i'd like the input autofill with zeros not provided to fill the mask to 02:00.
Is it possible?

**Comment by pyetrosafe** [comment]
Hi, I've made a code that might work for you.
Starting from a code to mask money, I changed it and I came up with one that will fill in automatically with zeros when I leave the field.

It also checks if the value entered is in accordance with hours and minutes, blocking the time up to 23, and minutes up to 59 ...

Code here:   https://jsfiddle.net/t0fvknpv/2/

Excuse possible English errors. Translated from Portuguese by google.
